Content.Server/Ghost/GhostCommand.cs

```diff
@@ -18,6 +19,8 @@ public sealed class GhostCommand : IConsoleCommand
         public void Execute(IConsoleShell shell, string argStr, string[] args)
         {
             var player = shell.Player;
+            var ticker = _entities.System<GameTicker>();
+
```

Comment:
We have a system for this now.

Reply:
https://github.com/space-wizards/RobustToolbox/pull/5267

MindSystem also needs updating to use it.